### DISCUSSION TITLE
Create Proposal bugs

### DISCUSSION THREAD

**Discussion by arcturiano** [BitCannaGlobal/WeedWallet#83] [General]
Bugs when creating a text proposal.

1. After creating a text proposal, I see the notification saying my proposal has been created and the "Click here" to see the proposal, but the "Click here" is not clickable/not a link. 

2. My proposal title, text and deposit remains the same, (does not get deleted after submitting" and "Sumbit Proposal" remains clickable after submitting too, so I can create an exactly same proposal right away, a possible proposal spam problem.  I created two exact proposal without a problem and without leaving the same create proposal screen.

3. I also did not see a Keplr transaction confirmation after submitting the proposals.  Samll pop up confirming transaction after accepting gas fees for proposal.     

![ClickHereNotWorking](https://github.com/BitCannaGlobal/WeedWallet/assets/94879750/ac5fd83b-73d6-4492-9f31-5b057788d82b)

**Comment by TJ-BitCanna** [comment]
Thanks for testing!

We'll take these with us!

**Comment by TJ-BitCanna** [comment]
Suggested issues:
- create hyperlink in the "click here" message after submitting a proposal
- clear fields after submitting a proposal
- create pop-up similar to all other transactions in the WeedWallet after submitting a proposal

**Comment by TJ-BitCanna** [comment]
The proposal page will be with limited access at the start. Over time it may be opened up for more people.
